.github/workflows/go.yml

```diff
@@ -21,5 +25,20 @@ jobs:
     - name: Build
       run: go build -v ./...
 
+    - uses: google-github-actions/setup-gcloud@v2
+      with:
+        version: '>= 486.0.0'
+        install_components: 'cloud-firestore-emulator'
+
+    - name: Start Firestore emulator
```

Comment:
Oh, huh, [`jobs` run in parallel by default](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-jobs-in-a-workflow#overview)!

Do we need to [impose an order](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-jobs-in-a-workflow#defining-prerequisite-jobs) on these? Will the emulator keep running in the background even after the job is done?

If the answer is "we need to submit this and try it to see", that's fine :D

Reply:
There's already an order imposed because *steps* run sequentially even though their containing *jobs* run in parallel. (The YAML diff doesn't make this very easy to see, but I only appended to the one existing job.)

That said, I *did* consider splitting `jobs.build` and `jobs.test`. But `go build` is so fast that all it would really do is cause `setup-go` module-cache issues between the two jobs.